service-tests/jetbrains.js

```diff
@@ -3,31 +3,53 @@
 const Joi = require('joi');
 const ServiceTester = require('./runner/service-tester');
 const { isMetric } = require('./helpers/validators');
+const { isSemver } = require('./helpers/validators');
 
-const t = new ServiceTester({ id: 'jetbrains', title: 'JetBrains plugin' });
+const t = new ServiceTester({ id: 'jetbrains', title: 'JetBrains plugin/version' });
 module.exports = t;
 
 t.create('downloads (number as a plugin id)')
   .get('/plugin/d/7495.json')
   .expectJSONTypes(Joi.object().keys({ name: 'downloads', value: isMetric }));
 
+t.create('jetbrains plugin (number as a plugin id)(v)')
+  .get('/plugin/v/7495.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'jetbrains plugin', value: isSemver }));
+
 t.create('downloads (plugin id from plugin.xml)')
   .get('/plugin/d/org.intellij.scala.json')
   .expectJSONTypes(Joi.object().keys({ name: 'downloads', value: isMetric }));
 
+t.create('downloads (plugin id from plugin.xml)(v)')
+  .get('/plugin/v/org.intellij.scala.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'jetbrains plugin', value: isSemver }));
+
 t.create('downloads (user friendly plugin id)')
   .get('/plugin/d/1347-scala.json')
   .expectJSONTypes(Joi.object().keys({ name: 'downloads', value: isMetric }));
 
+t.create('downloads (user friendly plugin id)(v)')
+  .get('/plugin/v/1347-scala.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'jetbrains plugin', value: isSemver }));
+
 t.create('unknown plugin')
   .get('/plugin/d/unknown-plugin.json')
   .expectJSON({ name: 'downloads', value: 'not found' });
 
+t.create('unknown plugin(v)')
```

Comment:
What do you think about `version for unknown plugin`?

Reply:
Fixed